infra/base-images/base-runner/run_fuzzer

```diff
@@ -132,6 +132,12 @@ if [[ "$FUZZING_ENGINE" = afl ]]; then
 
   CMD_LINE="$OUT/afl-fuzz $AFL_FUZZER_ARGS -i $CORPUS_DIR -o $FUZZER_OUT $(get_dictionary) $* -- $OUT/$FUZZER"
 
+  # TODO: come up with a better solution for indicating when a
+  # fuzzer is not in persistent mode.
+  if [[ "$FUZZER" == *"non_persistent"* ]]; then
+    CMD_LINE="$CMD_LINE @@"
```

Comment:
are all non-persistent targets called like this: `./target inputfile`? without any other command line options? If not maybe this needs a `target.nonpersistent" config file that defines how the target is to be called.
and note that afl++ can also send on stdin

Reply:
Yeah exactly, I'm going to make an API for doing this in the .options file we already support.

Reply:
Why don't we always add the @@ here? That way we don't need to scan the binary, oss fuzz doesn't have to care

Reply:
a) because otherwise it is not needed, b) I am not sure if the current commit on oss-fuzz of afl++ is one that works fine, or is an older one that has that strong performance decrease otherwise. I want to push an update, but not before next week.

Reply:
Understood.
I'm just saying, not having a grep on a potentially Chrome-sized binary vs adding an needless commandline flag may be worth it
Of course only makes sense of the target still works then (latest AFL++ version). Maybe keep it in mind :)